pulp_ansible/app/tasks/utils.py

```diff
@@ -216,3 +218,16 @@ def get_file_obj_from_tarball(tar, file_path, artifact_path, raise_exc=True):
         raise FileNotFoundError(f"{file_path} not found")
 
     return file_obj
+
+
+def get_last_sync_task(pk):
+    task = (
+        Task.objects.filter(reserved_resources_record__icontains=pk, name__icontains="sync")
+        .order_by("-pulp_created")
+        .first()
+    )
+
+    if task:
+        return TaskSerializer(task, context={"request": None}).data
```

Comment:
I'm wondering if we can make this function a property on the model called `last_sync_task` and then define the serializer field like `last_sync_task = TaskSerializer`.

https://www.django-rest-framework.org/api-guide/serializers/#dealing-with-nested-objects